bin/yugabyted

```diff
@@ -566,6 +584,28 @@ class ControlScript(object):
                 (_, was_started) = self.maybe_start_yw(is_first_run, is_first_install)
                 should_callhome = should_callhome or was_started
 
+            if is_first_install and not join_ip:
+                if self.get_failed_node_processes():
```

Comment:
can we move this functionality to a separate function like `process_init_env_vars` or similar. as we add init sql etc it will get longer.